packages/related-keyphrase-suggestions/src/components/KeyphrasesTable/index.js

```diff
@@ -0,0 +1,147 @@
+	const intentInitials = intent ? intent.split( "," ) : [];
+	const chartPoints = trend.split( "," ).map( value => parseFloat( value ) );
+
+	return (
+		<Table.Row>
+			<Table.Cell>
+				{ relatedKeyphrase }
+			</Table.Cell>
+			<Table.Cell>
+				<div className="yst-flex yst-gap-2">
+					{ intentInitials.length > 0 && intentInitials.map( ( initial, index ) => (
+						<IntentBadge key={ `${ index }-${ relatedKeyphrase }-${ initial }` } initial={ initial } />
+					) ) }
+				</div>
+			</Table.Cell>
+			<Table.Cell>
+				{ volume }
+			</Table.Cell>
+			<Table.Cell>
+				<TrendGraph data={ chartPoints } />
+			</Table.Cell>
+			<Table.Cell>
+				<DifficultyBullet value={ Number( difficulty ) } />
+			</Table.Cell>
+			{ isPremium && <Table.Cell className="yst-flex yst-justify-center yst-w-[124px]">
+				<TableButton onAdd={ noop } onRemove={ noop } />
+			</Table.Cell> }
+		</Table.Row>
+	);
+};
+
+KeyphrasesTableRow.propTypes = {
+	data: PropTypes.array.isRequired,
+	isPremium: PropTypes.bool,
+};
+
+/**
+ * Laoding keyphrase table row.
+ * @param {boolean} isPremium Whether the user is premium or not.
+ *
+ * @returns {JSX.Element} The loading row.
+ */
+const LoadingKeyphrasesTableRow = ( { isPremium = false } ) => {
+	return (
+		<Table.Row>
+			<Table.Cell className="yst-w-56">
+				<SkeletonLoader as="span" className="yst-w-[167px] yst-h-5" />
+			</Table.Cell>
+			<Table.Cell>
+				<SkeletonLoader className="yst-w-5 yst-h-5" />
+			</Table.Cell>
+			<Table.Cell>
+				<SkeletonLoader className="yst-w-[54px] yst-h-5" />
+			</Table.Cell>
+			<Table.Cell>
+				<SkeletonLoader className="yst-w-[66px] yst-h-5" />
+			</Table.Cell>
+			<Table.Cell>
+				<div className="yst-flex yst-gap-2">
+					<SkeletonLoader className="yst-w-[17px] yst-h-5" />
+					<SkeletonLoader className="yst-w-[11px] yst-h-5" />
+				</div>
+			</Table.Cell>
+			{ isPremium &&
+			<Table.Cell>
+				<SkeletonLoader className="yst-w-16 yst-h-7" />
+			</Table.Cell>
+			}
+		</Table.Row>
+	);
+};
+
+LoadingKeyphrasesTableRow.propTypes = {
+	isPremium: PropTypes.bool,
+};
+
+/**
+ *
+ * @param {object} rows The rows to display in the table.
+ * @param {boolean} isPremium Whether the user is premium or not.
+ *
+ * @returns  {JSX.Element} The keyphrases table.
+ */
+const KeyphrasesTable = ( { rows, isPremium = false } ) => {
+	return <Table>
+		<Table.Head>
+			<Table.Row>
+				<Table.Header>
+					{ __( "Related keyphrase", "wordpress-seo" ) }
+				</Table.Header>
+				<Table.Header>
+					{ __( "Intent", "wordpress-seo" ) }
+				</Table.Header>
+				<Table.Header>
+					{ __( "Volume", "wordpress-seo" ) }
+				</Table.Header>
+				<Table.Header>
+					{ __( "Trend", "wordpress-seo" ) }
+				</Table.Header>
+				<Table.Header className="yst-w-24">
+					{ __( "Keyphrase difficulty %", "wordpress-seo" ) }
+				</Table.Header>
+				{ isPremium && <Table.Header className="yst-text-right yst-w-20">
+					{ __( "Add keyphrase", "wordpress-seo" ) }
```

Comment:
After speaking with UX, since the fist header of the table is "Related Keyphras" it should be clear that "Add keyphrase" refer to the related kephrases. @uxkai

Reply:
Confirmed it's resolved

Reply:
Kind of strange if the action is `Remove`, no?